### PR TITLE
.coafile : Fix Wrong Command

### PR DESCRIPTION
**Pull request by vibhanshuvaibhav**
Changed the wrong command in .coafile from 
`default_action: *: ShowPatchAction` --> 
`default_actions = *: ShowPatchAction`
which is the correct command.

Closes https://github.com/coala/coala/issues/5043

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by ishanSrt** [comment]
@vibhanshuvaibhav why close this PR?

**Comment by vibhanshuvaibhav** [comment]
@ishanSrt I am new with Git and I felt something went wrong so thought of maybe restarting again. The issue is just a change of code but I yet unsure if I'm following the proper community guidelines.

**Comment by ishanSrt** [comment]
@vibhanshuvaibhav there is no need to close PRs. You can make the changes over here and keep adding to your PR 😀. Start by following the commit guidelines over here https://api.coala.io/en/latest/Developers/Writing_Good_Commits.html
You can also check out http://api.coala.io/en/latest/Developers/Git_Basics.html for a quick git tutorial. What you need to do here is use `git commit --amend` and then force push

**Comment by ishanSrt** [comment]
Also I edited the issue a little bit. This is not a typo but just a wrong way to use coala. BTW have you gone through the newcomer's guide

**Comment by vibhanshuvaibhav** [comment]
@ishanSrt what is CircleCI and TravisCI? It says tests failed on CircleCI, what does that mean?

**Comment by ishanSrt** [comment]
Please go throughly through the newcomer's guide and the docs. Everything's over there 👍http://api.coala.io/en/latest/Developers/Adding_CI.html
It tells you inconsistencies in your code. You can click details and see the error log

**Comment by vibhanshuvaibhav** [comment]
@ishanSrt I'll go through the newcomer's guide again but I never went through the page you mentioned. Thanks for that.

**Comment by ishanSrt** [comment]
I would recommend going through everything in the coala docs https://docs.coala.io/en/latest/ and everything under `For Developers` in https://api.coala.io/en/latest/ except for the api itself

**Comment by vibhanshuvaibhav** [comment]
I'll definitely go through everything you've asked me to. 
Thanks.

**Comment by ishanSrt** [comment]
remove the extra spaces inside the backticks. Since this is not a bug or a typo, use Closes instead of Fixes

**Comment by vibhanshuvaibhav** [comment]
Done.

**Comment by stobis** [comment]
> Changed the wrong command on line 25 of .coafile from
`default_action: *: ShowPatchAction` -->
`default_action = *: ShowPatchActio**n**`
which is the correct command.
> Closes #5043

You have a typo in commit message (missing n). Also, I'm not sure that describing a number of a changed line is necessary (but maybe someone more experienced can give a tip here :)).

unack e6189ba

File: .coafile [hunk 1]
```diff
@@ -22,7 +22,7 @@ default_actions = *: ApplyPatchAction
 # Do not set default_action to ApplyPatchAction as it may lead to some
 # required imports being removed that might result in coala behaving weirdly.
 
-default_action: *: ShowPatchAction
+default_action = *: ShowPatchAction
```
Comment (Makman2) [inline .coafile]: isn't it actually `default_actions`? (note the `s`)

**Comment by Makman2** [comment]
unack e6189ba

File: .coafile (hunk 1 above)
Comment (ishanSrt) [reply .coafile]: @Makman2 it works both ways https://github.com/coala/coala/blob/b5d7973a73badda75ee362e2b3c25df3b0226db6/.coafile#L25

Comment (ishanSrt) [reply .coafile]: although didn't test what would happen if more than one action is given without the `s`

**Comment by ishanSrt** [comment]
Squash your 2 commits into 1

**Comment by vibhanshuvaibhav** [comment]
@ishanSrt @Makman2 Is everything okay now?

**Comment by vibhanshuvaibhav** [comment]
@newbazz Removed 'on line 25'. Anything else that you would like to add?

**Comment by newbazz** [comment]
@vibhanshuvaibhav nope ;) LGTM

**Comment by Makman2** [comment]
ack 80f347f

**Comment by Makman2** [comment]
@gitmate-bot rebase

**Comment by Makman2** [comment]
@gitmate-bot ff

**Pull request merged**